components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/AutoSave.kt

```diff
@@ -229,6 +230,9 @@ private class StateMonitoring(
         } else if (lastObservation!!.loading != observation.loading && observation.loading == false) {
             autoSave.logger.info("Save: Load finished")
             true
+        } else if (lastObservation!!.inProgressMediaTabId != observation.inProgressMediaTabId) {
```

Comment:
I think this is probably too aggressive and we should maybe add later as needed? If a media is played for 30s then save would happen anyway, which seems good enough i.e. when force closing the app after a few seconds of playing the media probably doesn't need to work for this feature?